index.html

```diff
@@ -328,7 +328,10 @@ <h2 class="section-title">Sponsors</h2>
 				<h3>Silver Level Sponsors</h3>
 	      <a href="http://cdk-global.com" target="_blank"><img src="assets/img/logos/cdk-logo.png" /></a>
 	      <p><a href="http://cdk-global.com" target="_blank">CDK Global</a> is the largest global provider of integrated information technology and digital marketing solutions to the automotive retail industry.</p>
-	    </div>
+	    	<br/>
+				<a href="http://kcura.com" target="_blank"><img src="assets/img/logos/kcura-logo.jpg" /></a>
```

Comment:
Here I think the C in the image link has to be capitalized because the image isn't displaying on the page.